### PR TITLE
Add polyline support

### PR DESCRIPTION
**Pull request by davids410**
added `src/lit-google-map-polyline.ts` and added that to exports in index. rest of changes are related to formatting.

### CODE REVIEW

**Review by ElijahBrandyberry — APPROVED** [review]
This looks great! Only comment would be, we will likely need to rollback the style changes before we could merge it into the source repo

**Comment by davids410** [comment]
> This looks great! Only comment would be, we will likely need to rollback the style changes before we could merge it into the source repo

i thought it might be time for them to add styles to their repo. there are missing semicolons and inconsistent spacing, but i'm unsure of the etiquette in situations like this and would definitely defer to you.

**Comment by ElijahBrandyberry** [comment]
Let go for it, then!

**Pull request merged**
